vm-setup/roles/packages_installation/tasks/ubuntu_required_packages.yml

```diff
@@ -0,0 +1,101 @@
+---
+- name: Install required Ubuntu packages
+  block:
+    - name: Update apt
+      apt: update_cache=yes name=docker-ce state=latest
+
+    - name: Fetch yarn gpg key
+      apt_key:
+        url: https://dl.yarnpkg.com/debian/pubkey.gpg
+        state: present
+
+    - name: Add yarn release key
+      lineinfile:
+        path: /etc/apt/sources.list.d/yarn.list
+        line: deb https://dl.yarnpkg.com/debian/ stable main
+        create: yes
+        owner: root
+        group: root
+        mode: '0644'
+
+    - name: Add OS release key
+      lineinfile:
+        path: /etc/apt/sources.list.d/devel:kubic:libcontainers:stable.list
+        line: "deb https://download.opensuse.org/repositories/devel:/kubic:/libcontainers:/stable/xUbuntu_{{ OS_VERSION_ID }}/ /"
+        create: yes
+        owner: root
+        group: root
+        mode: '0644'
+
+    - name: Fetch OS release key
+      apt_key:
+        url: "https://download.opensuse.org/repositories/devel:/kubic:/libcontainers:/stable/xUbuntu_{{ OS_VERSION_ID }}/Release.key"
+        state: present
+
+    - name: Update apt
+      apt: update_cache=yes name=docker-ce state=latest
+
+    - name: Set update-alternatives to python3
+      community.general.alternatives:
+        name: python
+        path: /usr/bin/python3.6
+        link: /usr/bin/python
+      when: OS_VERSION_ID == "18.04"
+
+    - name: Set update-alternatives to python3
+      community.general.alternatives:
+        name: python
+        path: /usr/bin/python3.8
+        link: /usr/bin/python
+      when: OS_VERSION_ID != "18.04"
+
+    - name: Podman
+      block:
+        - name: Install podman
+          apt: name=podman
+          
+        - name: Default registries configuration file
+          blockinfile:
+            path: /etc/containers/registries.conf
+            block: |
+              [registries.search]
+              registries = ['docker.io', 'quay.io']
+      become: yes
+      when: CONTAINER_RUNTIME == "podman"
+
+    - name: Add Docker’s GPG key
+      apt_key:
+        url: https://download.docker.com/linux/ubuntu/gpg
+        state: present
+
+    - name: Add Docker Repository
+      apt_repository:
+        repo: deb https://download.docker.com/linux/ubuntu bionic stable
```

Comment:
Would it be bionic also in case of `focal`? Shouldn't we variablize this?

Reply:
Good point. I can make it parameterized to check what is the current OS release. Something like `$(lsb_release -cs)`